Gemfile

```diff
@@ -0,0 +1,100 @@
+source 'https://rubygems.org'
+ruby '3.0.1'
```

Comment:
@Goose97 let's use the template#develop branch instead of `the main` branch so we will have the latest Ruby and Rails version 

```
rails new <app_name> -m https://raw.githubusercontent.com/nimblehq/rails-templates/develop/template.rb
```